jekyll/_cci2/caching.md

```diff
@@ -214,6 +230,10 @@ Template | Description
 
 **Note:** When defining a unique identifier for the cache, be careful about overusing template keys that are highly specific such as {% raw %}`{{ epoch }}`{% endraw %}. If you use less specific template keys such as {% raw %}`{{ .Branch }}`{% endraw %} or {% raw %}`{{ checksum "filename" }}`{% endraw %}, you’ll increase the odds of the cache being used. But, there are tradeoffs as described in the following section.
```

Comment:
maybe this should be a section called Notes on Using Keys and Templates with a bulleted list at this point.

Reply:
Good call. Done!